File: java/src/main/java/deltix/luminary/implementation/FormalParameterNode.java
package deltix.luminary.implementation;

import java.util.List;

public class FormalParameterNode extends PropertyNode {
    private final boolean isParameterArray;

    protected FormalParameterNode(String name, String type, boolean isParameterArray, List<String> comments) {
        super(name, type, comments);
        this.isParameterArray = isParameterArray;
    }

    public boolean isParameterArray() {
        return isParameterArray;
    }
}

File: csharp/main/Implementation/FormalParameterNode.cs
using System;
using System.Collections.Generic;
using JetBrains.Annotations;

namespace Deltix.Luminary.Implementation
{
	internal class FormalParameterNode : PropertyNode
	{
		public FormalParameterNode([NotNull] String name, [NotNull] String type, Boolean isParameterArray, [CanBeNull] List<String> comments)
			: base(name, type, comments)
		{
			IsParameterArray = isParameterArray;
		}

		public Boolean IsParameterArray { get; }
	}
}
